admin_guide/manage_nodes.adoc

```diff
@@ -477,13 +477,123 @@ In rare cases, a router pod may not have a host port configured. In those cases,
 it is important to follow the xref:infrastructure-nodes[recommended restart
 process] for infrastructure nodes.
 
+[[modifying-nodes]]
+== Modifying Nodes
+
+// tag::node-configmap[]
+During installation, {product-title} creates a configuration map in the *openshift-node* project for each type of node group: 
+
+* *node-config-master* for xref:../architecture/infrastructure_components/kubernetes_infrastructure.adoc#master[master nodes];
+* *node-config-infra* for xref:../admin_guide/manage_nodes.adoc#infrastructure-nodes[infrastructure nodes];
+* *node-config-compute* for compute nodes.
+
+In order to make configuration changes to an existing node, edit the appropriate configuration map. A _sync pod_ on each node
+watches for changes in the configuration maps. The sync pods are automatically created by the installer using _sync daemonsets_ and
+create a *_/etc/origin/node/node-config.yaml_* file on each node, where the node configuration parameters reside. When a sync pod
+detects configuration map change, it updates the *_node-config.yaml_* on all nodes in that node group.
```

Comment:
After update the node-config.yaml, The sync pod also will kill the `hyperkube kubelet` process to trigger node restart use the updated configure file.